src/lib/chips/chip-list.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  ElementRef,
+  ModuleWithProviders,
+  NgModule,
+  QueryList,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {MdChip} from './chip';
+
+@Component(MdChipList.COMPONENT_CONFIG)
+export class MdChipList {
+
+  static COMPONENT_CONFIG: Component = {
```

Comment:
If this does work in AoT mode, then it should just be an exported constant, not a static member on the `MdChipList`

Either way, though, I'm tempted to say let's not worry about this for the initial version and add a TODO to explore the idea later.